### ISSUE TITLE
Feature request: Sign message for android

### ISSUE THREAD

**Issue by tbrannt** [Electron-Cash/Electron-Cash#903]
Is there a chance to have the sign(/verify) messages with private key feature available for the Android version of Electron Cash?

**Comment by mhsmith** [comment]
Yes, it's on the plan for the new replacement app. We're prioritizing matching the features of the existing app first, then the sign/verify feature will be added soon afterwards.

**Comment by tbrannt** [comment]
awesome:) thanks for the quick reply.
